eve/main.yml

```diff
@@ -254,6 +254,34 @@ models:
         tox -e tests -- -m \"${PYTEST_FILTERS}\""
       workdir: build/eve/workers/openstack-multiple-nodes/terraform/
       haltOnFailure: true
+  - ShellCommand: &add_final_status_artifact
```

Comment:
The commit message describe what you do, and this we can see with the diff, especially since the commit is small (and that's good :+1: )

But it doesn't explain why we add those helpers.

Reply:
Right, I've slightly changed the commit message, tell me if it's good for you now.

Reply:
Yup, LGTM now :+1:

Reply:
Cool, thanks.